ansible/utility/update-system.yml

```diff
@@ -0,0 +1,11 @@
+---
+- hosts:
+    kstest
+  become: true
+  become_user: root
+
+  tasks:
+  - name: Update all packages
+    dnf:
+      name: "*"
+      state: latest
```

Comment:
I don't think you need this playbook really. There is not much different call between:

`ansible-playbook -i hosts -k -K ./ansible/utility/update-system.yml`

and

`ansible kstest -i hosts -b -k -K  -m dnf -a "name=libguestfs-tools-c state=latest"`

Reply:
Hm, I think in this case reducing amount of stuff to type (and perhaps recall) can make a difference. I don't have problem having the playbook there if anyone finds it useful. It is just a utility.

Reply:
I agree with the last part. I don't see it useful but I don't have a problem to leave it there if anyone finds it useful.